test/testcafe/framework/page-objects/DeviceCodeActivatePageObject.js

```diff
@@ -71,4 +71,16 @@ export default class DeviceCodeActivatePageObject extends BasePageObject {
   isTryAgainButtonPresent() {
     return Selector('[data-se="try-again"]').exists;
   }
+
+  fillLegacyUserNameField(value) {
```

Comment:
make a separated file for v1. could live in `page-objects-v1`

Reply:
done